.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  'extends': 'airbnb',
+  'parser': 'babel-eslint',
+  'env': {
+    'jest': true,
+  },
+  'rules': {
+    'no-use-before-define': 'off',
```

Comment:
Why is this rule disabled? It usually is very handy for finding misspellings.

Reply:
I was following an example of setting up eslint airbnb after initializing a project and they describe the settings as the bare minimum. will correct :)